src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -178,8 +214,8 @@ export default function connectRatingMenu(renderFn, unmountFn) {
             instantSearchInstance,
             items: facetValues,
             hasNoResults: results.nbHits === 0,
-            refine: this._toggleRefinement,
-            createURL: this._createURL(state),
+            refine: toggleRefinement.bind(this, helper),
```

Comment:
It's not the same behaviour as before. Now between `init` & `render` we have two different functions. Previously the function was only created once at the `init` step. We do that to avoid to lost the reference inside the render function.